src/renderer/App.css

```diff
@@ -3,6 +3,8 @@
  *        See https://github.com/webpack-contrib/sass-loader#imports
  */
 
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@500&display=swap');
```

Comment:
I would usually install a font as a package from font-source (or other locations) when using. Happy to do it this way if it will not slow down production app. Does it download font during build or at runtime?

Reply:
Honestly not sure - I figured it might be at runtime but it's small so probably won't matter, and there's probably a caching layer somewhere. All very hand wavy, downloading the actual font sounds like a better idea

Reply:
Ok. As long as it works offline at runtime (as ours is specifically an offline app), then sounds good to me.